modules/client-configuration/pages/contact-methods-salt.adoc

```diff
@@ -1,14 +1,24 @@
 [[contact-methods-salt]]
 = Contact Methods for Salt Clients
 
-In most cases, Salt clients are registered accurately with the default bootstrap methods.
+The Salt contact method is the default choice and recommended unless there are specific needs.
+For more information about Salt in general, see xref:salt:salt-overview.adoc[].
+
+The Salt Contact Method is the best scaling method.
+All new {productname} features are supported and it has the widest variety of supported operating systems.
+All new operating systems are always supported with this contact method.
+
+Software updates are generally pushed from the server to the client.
+Connections are initiated from the client.
+This means you must open ports on the server, not on clients.
+The Salt clients are also known as Salt minions.
+{productname} Server (also known as Salt Server) installs a small daemon on every client.
```

Comment:
"Salt Server" or better "Salt master" ? I never heard "Salt Server".
and "installs a **small** daemon" is ... nice said, bit it is not small.

Reply:
ok.